sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -70,6 +70,7 @@ public partial class CapacityReservationData : Azure.ResourceManager.Models.Trac
         public string ProvisioningState { get { throw null; } }
         public string ReservationId { get { throw null; } }
         public Azure.ResourceManager.Compute.Models.ComputeSku Sku { get { throw null; } set { } }
+        public System.DateTimeOffset? TimeCreated { get { throw null; } }
```

Comment:
CreatedOn ?

Reply:
Looks like we have same name in 2 core libs already...

Reply:
I believe we could let this slide if the core libs had similar cases